nni/algorithms/compression/v2/pytorch/common/metrics_calculator.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Dict, List, Optional, Union
+
+import torch
+from torch import Tensor
+
+from nni.algorithms.compression.v2.pytorch.base.pruner_tools import MetricsCalculator
+
+__all__ = ['NormMetricsCalculator', 'MultiDataNormMetricsCalculator', 'DistMetricsCalculator',
+           'APoZRankMetricsCalculator', 'MeanRankMetricsCalculator']
+
+
+class NormMetricsCalculator(MetricsCalculator):
+    """
```

Comment:
Recommend adding annotation about pruner types this metrics_calculator serves.

Reply:
add it.